AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -2522,6 +2522,7 @@ function New-ADAccountForStorageAccount {
 
     Write-Verbose -Message "AD object name is $ADObjectName, SamAccountName is $SamAccountName."
 
+    $UserPrincipalName = "$spnValue@$Domain"

```

Comment:
nit: use lower case for the first character so that we know it is a local variable, not an input parameter

Reply:
updated